docs/cartesian-product.adoc

```diff
@@ -78,7 +78,7 @@ To demonstrate with a table:
 | 8th test  | "1"            | "1"            | "1"
 |===
 
-== Annotating your test method with @CartesianValueSource
+== Annotating your test method
 
 If you don't only supply string values to your test method like in the example above, you can annotate your method with `@CartesianValueSource`.
 `@CartesianValueSource` is used to define the possible inputs of a single test parameter - as annotations are listed top-to-bottom, they provide parameter values left-to-right.
```

Comment:
A bit below there is link to ValueSources of JUnit (can't comment that unchanged line), but the link only opens the user guide but not the anker of that sub topic

Reply:
https://junit.org/junit5/docs/current/user-guide/#writing-tests-parameterized-tests-sources-ValueSource This is the link. It absolutely _should_ open the user guide at the proper anchor.